docs/app/@theme/components/eva/eva.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="description">
+  <h2 class="h1 heading">
+    <span class="pre-heading">Based on</span>
+    Eva Design System
+  </h2>
+  <p class="text">Construct consistent & scalable interface using basic components following Eva Guidelines and it will always have stunning design. </p>
+  <a class="learn-more" nbButton size="giant">Learn more about Eva</a>
+</div>
+<div class="images">
+  <img class="theme-colors"
+       src="/assets/img/theme-colors.png"
```

Comment:
Seems that image srcs are not correct

Reply:
![image](https://user-images.githubusercontent.com/230527/58761841-29b3b880-8551-11e9-94e9-fec354cdf943.png)

Reply:
Fixed in 9f451b2